src/dependenciesChecks/index.js

```diff
@@ -40,17 +40,25 @@ module.exports = async (installInfo, testPath) => {
     }
   });
 
-  dependenciesResults.push({
-    metric: dependenciesMetrics.AXIOS_APISAUCE,
-    description: 'Esta instalado axios o apisauce en el proyecto',
-    value: packages.some(({ moduleName }) => moduleName === 'axios' || moduleName === 'apisauce')
-  });
-
   dependenciesResults.push({
     metric: dependenciesMetrics.JEST,
     description: 'Esta instalado Jest en el proyecto',
     value: packages.some(({ moduleName }) => moduleName === 'jest')
   });
 
+  if (tech === 'angular') {
+    dependenciesResults.push({
+      metric: dependenciesMetrics.NG_RX,
+      description: 'Esta instalado ngRx en el proyecto',
+      value: packages.some(({ moduleName }) => moduleName === '@ngrx/store')
+    });
+  } else {
+    dependenciesResults.push({
+      metric: dependenciesMetrics.AXIOS_APISAUCE,
+      description: 'Esta instalado axios o apisauce en el proyecto',
+      value: packages.some(({ moduleName }) => moduleName === 'axios' || moduleName === 'apisauce')
+    });
+  }
+
```

Comment:
The if is unnecesary. You can always push the result, and then in each summary we look for what we need.
In react not having `@ngrx/store` will not affect